lib/heartcheck/checks/redis.rb

```diff
@@ -16,6 +16,15 @@ def validate
         end
       end
 
+      def uri_info
```

Comment:
What about adding a comment to say that this method is used by `/inspect` endpoint?